lib/mutgen.c

```diff
@@ -840,7 +839,7 @@ mutgen_init_allocator(mutgen_t *self)
     /* Need to make sure we have enough space to store sites and mutations. We
      * allocate ancestral and derived states, as well as a list of mutations
      * for each site. This ensures that we can always allocate the required amount.
-     * We need to add one because the assert trips when the block size is equal
+     * We need to add one because the tsk_bug_assert trips when the block size is equal
```

Comment:
I'm not sure this is referring to `tsk_bug_assert`. I couldn't find which assertion it is referring to as removing the "plus ones" below had no adverse effect on the tests.

Reply:
Ah yes, you're right. This was working around a bug in an older version of tsk_blkalloc which has been fixed. Good spot.